test/test_jit_string.py

```diff
@@ -263,9 +264,10 @@ def test_rpartition():
         self.checkScript(test_rpartition, ())
 
         def test_split():
-            # type: () -> Tuple[List[str], List[str], List[str], List[str], List[str], List[str], List[str], List[str], List[str]]
+            # type: () -> Tuple[List[str], List[str], List[str], List[str], List[str], List[str], List[str], List[str], List[str], List[str]]
```

Comment:
Why not make this `List[List[str]]` so we don't have to keep editing it? Or even just delete the type comment since return types can be inferred